src/plugins/intel_cpu/src/nodes/convert.cpp

```diff
@@ -144,15 +168,16 @@ void Convert::execute(dnnl::stream strm) {
     if (parentPaddElemCount != childPaddElemCount)
         IE_THROW() << errorPrefix << " has different elements number in input and output buffers";
 
-    void* srcPtr = parentMem.GetPtr();
-    void* dstPtr = childMem.GetPtr();
+    std::vector<MemoryCPtr> srcMemory;
+    for (size_t i = 0; i < getOriginalInputsNumber(); i++) {
+        srcMemory.push_back(getParentEdgeAt(i)->getMemoryPtr());
+    }
+    std::vector<MemoryPtr> dstMemory;
+    for (size_t i = 0; i < getOriginalOutputsNumber(); i++) {
+        dstMemory.push_back(getChildEdgeAt(i)->getMemoryPtr());
+    }
```

Comment:
In prepare params we use only zero port, but here all the ports. May be it does make sense to do it uniformly?

Reply:
fixed